charts/optimize-live/templates/grafana/grafana-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if and .Values.grafana.enabled (or .Release.IsInstall .Values.grafana.forceGenerateConfigs) }}
```

Comment:
looking at `forceGenerateConfigs`, wondering if it might be good to ignore ones that controller changes/manages - i.e. so that a user can update dashboards without clearing out the datasources?  probably doesn't matter too much since controller would quickly change the datasources back, but just something i was curious about.

Reply:
It is a good idea. In this case, we will ignore `grafana-datasources` . will do.